enginetest/insert_queries.go

```diff
@@ -72,26 +71,6 @@ var InsertQueries = []WriteQueryTest{
 		SelectQuery:         "SELECT i FROM mytable WHERE s = 'x';",
 		ExpectedSelect:      []sql.Row{{int64(999)}},
 	},
-	{
```

Comment:
Where did these go?

Reply:
I know we have specific tests for prepared statements now, but I would just leave these where they are. It's fine to have overlapping coverage

Reply:
in a previous set of comments, you suggested that the analyzer should only apply bindings when we discover a prepared statement. I would need to revert those changes in order to put the prepared statement tests back in the regular test harness